ui/pages/bridge/transaction-details/bridge-step-list.tsx

```diff
@@ -85,7 +86,7 @@ export default function BridgeStepList({
             isEdgeComplete={isEdgeComplete}
           >
             <BridgeStepDescription
-              step={step}
+              step={step as Step}
```

Comment:
```suggestion
              step={step}
```


Do you need the type assertion here?

Reply:
Removed in [d49cbcf](https://github.com/MetaMask/metamask-extension/pull/28460/commits/d49cbcf83b1950a4de52c0cfb7388fb4fdbd8a8c)